aws/table_aws_globalaccelerator_endpoint_group.go

```diff
@@ -28,7 +28,7 @@ func tableAwsGlobalacceleratorEndpointGroup(_ context.Context) *plugin.Table {
 			KeyColumns: []*plugin.KeyColumn{
 				{Name: "listener_arn", Require: plugin.Optional},
 			},
-			ParentHydrate: listGlobalAcceleratorAccelerators,
+			ParentHydrate: listGlobalAcceleratorListeners,
```

Comment:
Note this defines `listGlobalAcceleratorListeners` as the parent hydrate function. The table definition for `aws_globalaccelerator_listener` in turns defines `listGlobalAcceleratorAccelerators ` as the parent hydrate for the `listGlobalAcceleratorListeners` hydrate function: https://github.com/nmische/steampipe-plugin-aws/blob/nested_parent_hydrate/aws/table_aws_globalaccelerator_listener.go#L31.